### PR TITLE
Expanse docs

### PR DESCRIPTION
**Pull request by gurtajkhatra-panw**
## Motivation and Context

Testing out changes to add Xpanse API docs

## How Has This Been Tested?
Locally

## Types of changes
Adding Docs
Updating the `sidebar.js` logic.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: static/oas/expander/entities.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+security:
+  - JWT: []
+info:
+  description: An API for interacting with data from Expanse's Expander product.
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:11]: Expanse -> Xpanse

File: static/oas/expander/entities.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  title: Expander API - Entities
+  version: 1.0.0
+  termsOfService: 'https://www.expanse.co/master-service-agreement/'
+  contact:
+    name: API Support
+    url: 'https://expander-support.expanse.co'
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:17]: these should be updated right?

File: static/oas/expander/entities.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+paths:
+  /IdToken/:
+    get:
+      summary: Get ID token
+      description: Returns the ID token to be used with the other api endpoints as the API token.
+      security: []
+      parameters:
+        - name: Authorization
+          in: header
+          description: 'Expanse-provided refresh token to be passed in Authentication header. Value must be prefixed with `Bearer`. Example: `Authorization: Bearer {your refresh token}`'
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:35]: Xpanse

File: static/oas/expander/entities.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+            type: object
+            required:
+              - token
+            properties:
+              token:
+                type: string
+  /Entity/:
+    get:
+      summary: List Entities
+      description: Returns the list of entities to which the authenticated user has access. WARNING! Fetching IP Ranges from this endpoint is now deprecated and may return incorrect data.
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:51]: We can remove the WARNING

File: static/oas/expander/entities.yml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        '200':
+          description: Entities list
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Entity'
+  /Entity/{entityId}/:
+    get:
+      summary: Entity Details
+      description: Returns the details for a given Entity. WARNING! Fetching IP Ranges from this endpoint is now deprecated and may return incorrect data.
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:82]: remove the WARNING

File: static/oas/expander/entities.yml (hunk 4 above)
Comment (vinaymayar) [reply static/oas/expander/entities.yml:51]: list of entities -> list of Business Units

File: static/oas/expander/entities.yml [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+          schema:
+            type: object
+            required:
+              - token
+            properties:
+              token:
+                type: string
+  /Entity/:
+    get:
+      summary: List Entities
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:50]: Maybe: List Business Units (aka Entities)

File: static/oas/expander/entities.yml [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+      description: Returns the list of entities to which the authenticated user has access. WARNING! Fetching IP Ranges from this endpoint is now deprecated and may return incorrect data.
+      parameters:
+        - name: name
+          in: query
+          required: false
+          type: string
+          description: Find an Entity by the given name
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:57]: Business Unit

File: static/oas/expander/entities.yml [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+        - name: parent
+          in: query
+          required: false
+          type: string
+          format: uuid
+          description: Find Entities with the given parent Entity
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:63]: Business Unit

File: static/oas/expander/entities.yml [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+        - name: has_parent
+          in: query
+          required: false
+          type: string
+          enum:
+            - 'True'
+            - 'False'
+          description: Find Entities that either have or do not have parents
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:71]: Business Unit

File: static/oas/expander/entities.yml [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+      responses:
+        '200':
+          description: Entities list
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:74]: Business Unit list

File: static/oas/expander/entities.yml (hunk 5 above)
Comment (vinaymayar) [reply static/oas/expander/entities.yml:82]: Entity -> Business Unit

File: static/oas/expander/entities.yml [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+      id:
+        type: string
+        format: uuid
+      name:
+        type: string
+      parent:
+        type: string
+        format: uuid
+        description: ID of parent Entity (may be null)
+      primarySector:
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:109]: remove this

File: static/oas/expander/issues.yml [hunk 12]
```diff
@@ -0,0 +1,2257 @@
+openapi: 3.0.1
+info:
+  title: Expander API - Issues
+  description: An API for interacting with Issues.
+  termsOfService: https://www.expanse.co/master-service-agreement/
+  contact:
```
Comment (vinaymayar) [inline static/oas/expander/issues.yml:6]: this needs to be updated

File: static/oas/expander/entities.yml [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+  title: Expander API - Entities
+  version: 1.0.0
+  termsOfService: 'https://www.expanse.co/master-service-agreement/'
+  contact:
+    name: API Support
+    url: 'https://expander-support.expanse.co'
+    email: expander-support@expanseinc.com
+securityDefinitions:
+  JWT:
+    description: 'For accessing the API a valid JWT token must be included in every query in the ''Authorization'' header. A valid JWT token is generated by the API and returned via the /IdToken/ route. The value for this header should be `JWT <token>`, where the token is the base64-encoded JSON Web Token. Example: `Authorization: JWT deadBEEF12345`'
```
Comment (vinaymayar) [inline static/oas/expander/entities.yml:21]: can we set a better example than `deadBEEF12345`? we use that in a lot of our APIs for some reason

File: static/oas/expander/services.yml [hunk 14]
```diff
@@ -0,0 +1,2151 @@
+# This file was generated by OpenAPIDocsIT::writeUpdatedOpenApi at 2021-04-30T21:03:07.961Z. Do not modify.
+openapi: 3.0.1
+info:
+  title: Expander Services API
+  description: API for example purposes
+  termsOfService: https://www.expanse.co/master-service-agreement/
+  contact:
```
Comment (vinaymayar) [inline static/oas/expander/services.yml:7]: this needs to be updated

File: static/oas/expander/services.yml [hunk 15]
```diff
@@ -0,0 +1,2151 @@
+# This file was generated by OpenAPIDocsIT::writeUpdatedOpenApi at 2021-04-30T21:03:07.961Z. Do not modify.
+openapi: 3.0.1
+info:
+  title: Expander Services API
+  description: API for example purposes
```
Comment (vinaymayar) [inline static/oas/expander/services.yml:5]: this needs to be updated

File: static/oas/expander/targeted-ips.yml [hunk 16]
```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.1
```
Comment (vinaymayar) [inline static/oas/expander/targeted-ips.yml:1]: are we sure that this API is an API that we want to expose to customers?

File: static/oas/expander/assets-v2-swagger.yml [hunk 17]
```diff
@@ -0,0 +1,4660 @@
...
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: include
+          in: query
+          description: 'Comma-separated string; Include the provided fields as part
```
Comment (vinaymayar) [inline static/oas/expander/assets-v2-swagger.yml:2369]: this is inaccurate and has been fixed recently. we should fetch the latest value of this file

**Review by vinaymayar — COMMENTED** [review]
a couple more comments! likely none of these are blockers, but could be things to follow up on

**Review by csestito — APPROVED** [review]

**Pull request merged**